recipes/graaf/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.11)
+project(test_package LANGUAGES CXX)
+
+find_package(Graaf REQUIRED CONFIG)
```

```suggestion
find_package(graaflib REQUIRED CONFIG)
```

we would need to update this here too